docs/source-pytorch/fabric/fabric.rst

```diff
@@ -414,9 +389,9 @@ To define your own behavior, subclass the relevant class and pass it in. Here's
 ----------
 
 
-************************
-Lightning Fabric Methods
-************************
+**************
+Fabric Methods
+**************
```

Comment:
here you may also need to add the separate methods for setting up only optimizer or only the model.

Reply:
Agreed, these will have to be added. PRs 3/n etc. will go section by section